instrumentation/p6spy/src/main/java/brave/p6spy/TracingJdbcEventListener.java

```diff
@@ -30,6 +34,7 @@
     // regardless of noop or not, set it in scope so that custom contexts can see it (like slf4j)
     if (!span.isNoop()) {
       String sql = includeParameterValues ? info.getSqlWithValues() : info.getSql();
+      if (sql == null) sql = QUERY_PLACEHOLDER;
       span.kind(Span.Kind.CLIENT).name(sql.substring(0, sql.indexOf(' ')));
```

Comment:
Could do, but we also need it the line below: `span.tag(TraceKeys.SQL_QUERY, sql);`